doc/quick-start/quick-start.md

```diff
@@ -1,5 +1,39 @@
 # Quick Start for Data Prep Kit 
-Here we provided short examples of various uses of the Data Prep Kit.
+Here we provided short examples of various uses of the Data Prep Kit. Most users who want to jump right in can use standard pip install to deploy the data-prep-kit and the python or ray transforms to their virtual python environment. 
+
+- When setting up a virtual environment it is recommended to use python3.11 as in the example below using conda:
+
+    - setup a virtual environment (example using conda) <br>
+        `conda create -n data-prep-kit-1 -y python=3.11`
+<br>
+
+    - activate the new conda environment<br>
+        `conda activate data-prep-kit-1`
+<br>
+
+    - make sure env is swithced to data-prep-kit-1 and Check python version. <br>
+        `python --version`
+        <br> The command above should say: should say : 3.11
+<br>
+
+    - install jupyter lab<br>
+        `pip3 install jupyterlab`   
+<br><br>
+
+- Deploy the latest release of the data prep toolkit library<br>
+    `pip3 install data-prep-toolkit`
+
+or 
+-  deploy the latest releases of the data prep toolkit library and all python transforms
+
+    `pip3 install data-prep-toolkit-transforms`
```

Comment:
I made this comment elsewhere, but just to locate it, this is not part of the `dev` branch  yet.  That is, the `transforms/packaging` directory needs to be present before we start referring to this wheel.

Reply:
That should be ok for now. the packaging PR still need some more documentation but let's not hold back this PR for it.